### DISCUSSION TITLE
Welcome to enpal-homeassistant Discussions!

### DISCUSSION THREAD

**Discussion by skipperro** [skipperro/enpal-homeassistant#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Busterdick** [comment]
Da Enpal das interface am 18.11. mit einem neuen Update versehen hat, sind nur noch 6 Entities abfragbar. es funktionieren nur noch sesor.enpal_energy_consumption,
sensor.enpal_energy_external_in_day,
sensor.enpal_energy_external_out_day,
sensor.enpal_power_external_total,
sensor.enpal_production_day,
sensor.enpal_solar_production_power

alle anderen Sensoren wie z.B. Battery und Wallbox gehen nicht mehr.

Wird es da wohl eine Anpassung geben?

Grüsse Andreas

**Comment by skipperro** [comment]
Ja, ich arbeite gerade dran und hoffe das ich heute 0.3.0 version fertig machen kann. Du kannst aktuelle test-version schon testen. In HACS aktievere Beta Versionen und dann mach den Download direkt aus "main" branch in GitHub:
![image](https://github.com/user-attachments/assets/a2182067-4cbc-4495-9c96-ca44b2a41a2f)

**Reply by Busterdick** [reply]
Cool, danke Dir. Das mit der beta bekomme ich nicht hin  ;-) zeigt mir nix an. Ist aber nicht schlimm. Wenn es die Tage fertig wird ist ja schon super :-)

**Reply by Busterdick** [reply]
Super!!! hab gesehen das Du die RC schon hochgeladen hast. Habe lediglich die Sensor.py und manifest.json "ausgetauscht" und nun gehts wieder. Dir lieben Dank für Deine Mühe! Top!!!!

**Comment by HealsCodes** [comment]
Es scheint so als ob es um den 25.11 rum weitere Updates gab und die InfluxDb nicht länger mit werten gefüttert wird (aktuell läuft auf unserer Box ` Solar Rel. 8.43.3+249078 (28.11.2024)`)

Über die ENPAL-eigene App werden Daten angezeigt und Über die Weboberfläche der Box kann man zumindest auch noch teils die aktuellen Daten anzeigen lassen, aber das `solar` Bucket der InfluxDB hat nach dem 25.11.2024 keine werte mehr (Historie vorher lässt sich aber abfragen).

Ist das ein Problem mit unserer Box oder sind wir da nicht die Einzigen?

**Comment by skipperro** [comment]
Bitte hier schauen: https://github.com/skipperro/enpal-homeassistant/issues/9
